Simplenote/SPAppDelegate+Extensions.swift

```diff
@@ -30,6 +30,21 @@ extension SPAppDelegate {
             bucket.delegate = self
         }
     }
+
+    @objc
+    func migrateCoreDataToAppGroupIfNeeded() {
+        guard let newDbURL = SharedStorageMigrator.newDbURL else {
+            return
+        }
+
+        // Confirm if the app group DB exists
+        if FileManager.default.fileExists(atPath: newDbURL.path) {
+            print("Core Data Migration already complete")
+            return
+        }
+
+        SharedStorageMigrator.migrateCoreDataToAppGroup()
```

Comment:
Ideally all of the logic to "migrate if needed" is encapsulated into `SharedStorageMigrator`.

Goal is for the migration details not to leak anywhere else, and keep all of the checks encapsulated.